### DISCUSSION TITLE
Map domain to coolify hosted service

### DISCUSSION THREAD

**Discussion by brianmichel** [coollabsio/coolify#2965] [Q&A]
Hey everyone! I'm new to self hosting in general so this might be a straightforward question to answer but I just can't figure it out myself.

I have a site successfully hosted at https://twofy.sunday.industries which is where my coolify instance is running. I purchased twofy.xyz that I'd like to redirect to twofy.sunday.industries so it's a bit easier to remember.

I have cloudflare cnames set up for twofy.xyz and www.twofy.xyz to route to twofy.sunday.industries however I keep getting a 404 on twofy.xyz.

I think this is because caddy isn't set up to handle twofy.xyz at all (only *.sunday.industries).

Is there a way I can get my coolify setup to also handle twofy.xyz?

**Comment by peaklabs-dev** [comment]
Just add https://twofy.xyz,https://twofy.sunday.industries to the domain so that the application is accessible on both domains (comma separated list).
